src/views/CallbackView.js

```diff
@@ -1,11 +1,15 @@
 import React from 'react';
+import {Helmet} from 'react-helmet';
 import {auth} from '../state/auth';
 import {Container, Segment, Placeholder} from 'semantic-ui-react';
 
 const CallbackView = ({history}) => {
   auth.handleAuthentication(history);
   return (
     <>
+      <Helmet>
+        <title>KF Data Tracker - Forwarding</title>
+      </Helmet>
```

Comment:
Is this necessary as this view is just flashed to the user?

Reply:
This would be helpful in debugging if users ever stuck at the this step or it takes too long.